tools/mdanalysis/rdf.xml

```diff
@@ -0,0 +1,92 @@
+<tool id="mdanalysis_rdf" name="RDF Analysis" version="@VERSION@">
+    <description>Radial Distribution Function between two atoms</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>   
+    <expand macro="requirements" />
+    <command detect_errors="exit_code">
+<![CDATA[
+    python '$__tool_directory__/rdf.py' 
+        --idcd '$dcdin' 
+        --ipdb '$pdbin' 
+        --isegid1 '$segid1'
+        --iresid1 '$resid1'
+        --iname1 '$name1' 
+        --isegid2 '$segid2'
+        --iresid2 '$resid2'
+        --iname2 '$name2'
+        --inbins '$nbins'
+        --istart '$start' 
+        --iend '$end' 
+        --output '$output' 
+        --ordf_plot '$rdf_plot'
+    2>&1
+]]></command>
+    <inputs>
+        <param format="dcd" name="dcdin" type="data" label="dcd trajectory input"/>
+        <param format="pdb" name="pdbin" type="data" label="pdb input"/>
+        <param name="segid1"  type="text" value="PRO" label="Segid of atom 1"/>
+        <param name="resid1"  type="text" value="212" label="Resid of atom 1"/>
+        <param name="name1"  type="text" value="OE2" label="Atom name of atom 1"/>
+        <param name="segid2"  type="text" value="HET" label="Segid of atom 2"/>
+        <param name="resid2"  type="text" value="3" label="Resid of atom 2"/>
+        <param name="name2"  type="text" value="C1" label="Atom name of atom 2"/>
+        <param name="nbins"  type="text" value="100" label="Number of bins in the histogram"/>
+        <param name="start"  type="text" value="0.0" label="Starting Point"/>
+        <param name="end"  type="text" value="5.0" label="End Point"/>
+    </inputs>
+    <outputs>
+        <data format="csv" name="output" label="RDF raw data"/>
```

Comment:
This should be `tabular` not `csv`. csv is usually really comma-separated-files ...
Can you change this here and in the other tools?